src/csharp/Microsoft.Spark/Sql/DataFrameReader.cs

```diff
@@ -34,7 +35,25 @@ public DataFrameReader Format(string source)
             _jvmObject.Invoke("format", source);
             return this;
         }
-
+        
+        /// <summary>
+        /// Specifies the schema by using StructType object.
```

Comment:
Let's try to enrich the comments
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/cref-attribute

```suggestion
        /// Specifies the schema by using <see cref="StructType"/>.
```

Reply:
Sounds good. Thanks.